app/views/calculators/adjustable_income/_calculator.html.erb

```diff
@@ -0,0 +1,2 @@
+<%= render 'form', form: form %>
+<%= render 'estimate', estimate: form.estimate, pot: form.pot, monthly_drawdown_amount: form.estimate[:monthly_drawdown_amount] if form.valid? %>
```

Comment:
The postfix modifier is a surprise here since the line is reasonably long. I think this could be clearer at a glance if we move the `render` call inside the conditional instead.